Opcodes/physmod.c

```diff
@@ -35,6 +35,20 @@
 #include "brass.h"
 #include <math.h>
 #include "interlocks.h"
+
+static inline double intpow_(MYFLT x, uint32_t n)
```

Comment:
Did a search and we have intpow, intpow_, intpowint, and intpow1 in our codebase. Think we could consolidate all of these further into one function.

Reply:
reduced to two inline functions.